internal/painter/software/draw.go

```diff
@@ -7,9 +7,9 @@ import (
 	"fyne.io/fyne"
 	"fyne.io/fyne/canvas"
 	"fyne.io/fyne/internal"
-	"fyne.io/fyne/internal/painter"
+	"fyne.io/fyne/internal/cache"
+	pnt "fyne.io/fyne/internal/painter"
```

Comment:
"pnt"? why did this move from "painter"? If it has to the name could be something more clear like "paint" or "intPainter"